content/client-lib-development-guide/features.textile

```diff
@@ -339,6 +340,14 @@ h3(#rest-encryption). Encryption
 h3(#rest-compatibility). Forwards compatibility
 * @(RSF1)@ The library must apply the "robustness principle":https://en.wikipedia.org/wiki/Robustness_principle in its processing of requests and responses with the Ably system. In particular, deserialization of Messages and related types, and associated enums, must be tolerant to unrecognised attributes or enum values. Such unrecognised values must be ignored.
 
+h3(#batch-operations). Batch Operations
+* @(BO1)@ The batch operations functions must use the REST endpoints in Batch Mode, sending a single request containing all specified data
+* @(BO2)@ Batch operations must be able to be performed for the following:
```

Comment:
I know you've got method names in the IDL but I think generally in the spec we mention the name of the method in the spec too, see RSE1 and RSP3 for examples.

Reply:
[Done](https://github.com/ably/docs/pull/1421/commits/2cdba4a3cc09efeb703bacc864b0e1f6489e068a)